thoth/storages/graph/models.py

```diff
@@ -1688,6 +1687,18 @@ class SIAggregated(Base, BaseExtension):
     python_package_version_entity = relationship("PythonPackageVersionEntity", back_populates="si_aggregated_runs")
 
 
+class ImportPackages(Base, BaseExtension):
+    """Packages imported as a result of solver run"""
+
+    __tablename__ = "import_packages"
+
+    document_id = Column(Text, nullable=False)
```

Comment:
This can be obtained from `PythonPackageVersion` directly so no need to keep it here.

```suggestion
```